File: tagalong/016-maps/maps.py
ages = {
    "Sarah":        32,
    "Billy":        12,
    "Jeremiah":     99,
    "John Baptist": 47,
}
print(ages["Sarah"])
billyAge, billyPresent = 0, False
if "Billy" in ages:
    billyAge, billyPresent = ages["Billy"], True

print(billyAge, billyPresent)

x13Age, x13Present = 0, False
if "x13" in ages:
    x13Age, x13Present = ages["x13"], True

print(x13Age, x13Present)

ages["Faustus"] = 66
print(ages)

File: tagalong/016-maps/maps.go
package main

import "fmt"

func main() {
	ages := map[string]int{
		"Sarah":        32,
		"Billy":        12,
		"Jeremiah":     99,
		"John Baptist": 47,
	}
	fmt.Println(ages["Sarah"])

	billyAge, billyPresent := ages["Billy"]
	fmt.Println(billyAge, billyPresent)

	x13Age, x13Present := ages["x13"]
	fmt.Println(x13Age, x13Present)

	ages["Faustus"] = 66
	fmt.Println(ages)
}
